gregoriotex/features.html

```diff
@@ -55,52 +55,91 @@
 
-<p>The red it will produce is quite close to the red of the scores I have been able to find, but it's not perfect. If you want to adapt it with your own color, call the macro <code>\coloredlines{}{}{}</code> that will take the RGB code (in decimal) of the color you want to give to the lines. Once again, call <code>\normallines</code> to cancel it. If you find a better red than the one of Gregorio, please suggest it!</p>
+<p>The red it will produce is quite close to the red of the scores I have been able to find, but it's not perfect.  If you want to adapt it with your own color, you can either define a new color or redefine <code>gregoriocolor</code> with <code>\definecolor</code>. See the documentation of the xcolor package for details on how to use this command.  If you find a better red than the one of Gregorio, please suggest it!</p>
 
-<h2 id="symbols">Symbols included in GregorioTeX</h2>
+<h2 id="symbols">Symbols included in Gregorio<span class="tex">T<span class="epsilon">e</span>X</span></h2>
 
-<p>GregorioTeX includes several symbols and some macros to call them. These symbols are very useful for scores and books of gregorian chant, and it's almost impossible to find them outside of GregorioTeX. They may even interest people wanting to make their books more beautiful. A lot of symbols are still missing, so any help or suggestion is welcome!</p>
+<p>Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> includes several symbols and some macros to call them. These symbols are very useful for scores and books of gregorian chant, and it's almost impossible to find them outside of Gregorio<span class="tex">T<span class="epsilon">e</span>X</span>. They may even interest people wanting to make their books more beautiful. A lot of symbols are still missing, so any help or suggestion is welcome!</p>
 
-<p>To get the symbols of GregorioTeX you don't need any special TeX system, you can simply use it with your default LaTeX or pdfLaTeX. Install GregorioTeX and include the package <code>gregoriosyms</code>.</p>
+<p>In documents where you want to use these symbols but don't need to include any scores, you can make use of the <code>gregoriosyms</code> pacakge instead of the <code>gregoriotex</code> package.  This lighter-weight package will only load the commands related to the symbols.</p>
 
-<h4>Simple symbols</h4>
+<h3>Simple symbols</h3>
 
-<p>Here are the simple symbols of GregorioTeX:</p>
+<p>Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> comes with a special font, greextra, which contains some symbols which appear in liturgical documents but which are difficult to find in other fonts.  Some examples of these are shown below:</p>
 
 <div class="legende">
-<img src="/illus/symbols.png" title="" alt="" />
+<img src="../illus/symbols.png" title="" alt="" />
```

Comment:
I tried to retain the original images where possible because it's easier than remaking them, but seeing as I have to revise several other images, it might make sense to revise this one and split it up.  Opinions?